includes/view/AngelTypes_view.php

```diff
@@ -477,6 +544,15 @@ function AngelType_view_table_headers(AngelType $angeltype, $supporter, $admin_a
         }
         $headers['ifsg_certificate'] = __('ifsg.certificate');
     }
+    if (auth()->can('angeltype.goodie.list') &&
```

Comment:
If you really want to format it that way it would be
```suggestion
    if (
        $goodie_enabled
        && auth()->can('angeltype.goodie.list')
        && auth()->can('user.goodie.edit')
    ) {
        $headers['goodie_actions'] = __('Goodie actions');
        if ($goodie_tshirt) {
            $headers['goodie_actions'] = __('T-shirt actions');
            $headers['shirt_size'] = __('user.shirt_size');
        }
    }
```